template/vue.config.js

```diff
@@ -17,7 +17,8 @@ module.exports = {
     // disable eslint
     config.module.rules.delete('eslint');
     config.module.rules.delete('svg');
-
+    config.module.rule('vue').uses.delete('cache-loader');
```

Comment:
Don't add this code to the theme. Add this is in CLI

Reply:
Done. @ZaidRehman